components/tooltip.js

```diff
@@ -0,0 +1,76 @@
+import { IconButton } from 'theme-ui'
+import { Info } from '@carbonplan/icons'
+
+import { Box, Flex } from 'theme-ui'
+import { useState } from 'react'
+import AnimateHeight from 'react-animate-height'
+
+const Tooltip = ({ expanded, setExpanded, sx }) => {
+  return (
+    <IconButton
+      onClick={() => setExpanded(!expanded)}
+      role='checkbox'
+      aria-checked={expanded}
+      aria-label='Information'
+      sx={{
+        cursor: 'pointer',
+        height: '16px',
+        width: '16px',
+        '@media (hover: hover) and (pointer: fine)': {
+          '&:hover > #info': {
+            stroke: 'primary',
+          },
+        },
+        p: [0],
+        transform: 'translate(0px, -3.75px)',
+        ...sx,
+      }}
+    >
+      <Info
+        id='info'
+        sx={{
+          stroke: expanded ? 'primary' : 'secondary',
+          transition: '0.1s',
+        }}
+      />
+    </IconButton>
+  )
+}
+
+export const TooltipWrapper = ({
+  children,
+  tooltip,
+  mt = '8px',
+  color,
+  sx,
+}) => {
+  const [expanded, setExpanded] = useState(false)
+
+  return tooltip ? (
+    <>
+      <Flex
+        sx={{
+          justifyContent: 'space-between',
```

Comment:
Documenting Slack discussion: on large screens, the space can become quite large, which makes it easy to miss the tooltip. Rendering the tooltip to the left of `children` with a fixed `gap` should be an easy fix!
![image](https://github.com/carbonplan/leap-data-catalog/assets/12436887/ca81e383-4f6e-43d4-97df-68dd0b0441e5)